### PR TITLE
Issue 1309; give executable permissions to /dev/shm for standalone co…

### PR DESCRIPTION
**Pull request by theddub**
…ntainer

**Thanks for contributing to the Docker-Selenium project!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines, applied for this repository.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Standalone/Dockerfile.txt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 USER 1200
 
+RUN sudo chmod +x /dev/shm
```
Comment (diemol) [inline Standalone/Dockerfile.txt]: This should be done in the `NodeBase` image

**Review by diemol — CHANGES_REQUESTED** [review]

File: Standalone/Dockerfile.txt (hunk 1 above)
Comment (theddub) [reply Standalone/Dockerfile.txt]: Okay, I've updated it with the change in NodeBase image and I tested it there and it works if you want to look at this PR updated.

Comment (theddub) [reply Standalone/Dockerfile.txt]: @diemol It's now in NodeBase Dockerfile, I'm not in love with my comment I made in the code....I was trying to follow your Dockerfile "comments pattern" you guys were doing, feel free to change as you'd like.

Comment (diemol) [reply Standalone/Dockerfile.txt]: I see it in `Base`, not in `NodeBase`, am I missing something?

Comment (theddub) [reply Standalone/Dockerfile.txt]: Sorry, I didn't pay close enough attention and I did put in in Base, putting it in NodeBase now, then I'll test it again and update the PR.

Comment (theddub) [reply Standalone/Dockerfile.txt]: @diemol Fix is now in NodeBase, I tested that it works. Feel free to change my code comment in NodeBase.

Comment (theddub) [reply Standalone/Dockerfile.txt]: @diemol anything else needed? I think this is all good now?

Comment (diemol) [reply Standalone/Dockerfile.txt]: @theddub before merging this, did you check if the image size increased? If so, how much?

Comment (theddub) [reply Standalone/Dockerfile.txt]: @diemol I just checked, base(obviously not effected now), node-base, node-chrome & standalone-chrome all stayed exactly the same as reported( to the 1/100 of a MB as seen in Docker Desktop...ie...doesn't round to the nearest MB like "docker image ls" does) by Dokcer(docker image ls) and Dokcer Desktop(GUI). No change in image size whatsoever from before to after.

**Review by diemol — APPROVED** [review]
Thank you, @theddub!

**Comment by theddub** [comment]
> Thank you, @theddub!

Thank you @diemol for taking the time to investigate the issue and for all your help and input!

**Pull request merged**